src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -492,12 +502,20 @@ private void updateBytesInfo() {
         }
     }
 
+    private void putItemStackIntoMap(final ItemStackMap<Integer> map, final ItemStack stack, final int cellStatus) {
+        ItemStack newStack = new ItemStack(stack.getItem(), 1, stack.getItemDamage());
+        if (map.computeIfPresent(newStack, (currentStack, stackCount) -> stackCount += 1) == null) {
```

Comment:
No need in assignment - you just need to return new value from this function.
```suggestion
        if (map.computeIfPresent(newStack, (currentStack, stackCount) -> stackCount + 1) == null) {
```